provisioner.go

```diff
@@ -344,6 +344,11 @@ func (p *LocalPathProvisioner) createHelperPod(action ActionType, cmdsForPath []
 							ReadOnly:  false,
 							MountPath: "/data/",
 						},
+						{
+							Name:      "script",
+							ReadOnly:  false,
+							MountPath: "/tmp",
```

Comment:
Don't use `/tmp`. It's a part of the helper pod anyway.

Reply:
i chaned /tmp to /script which is not a part of the helper pod.